rgw/v2/tests/s3cmd/test_rate_limit.py

```diff
@@ -10,6 +10,7 @@
 Polarion Tests:
 CEPH-83574910
 CEPH-83574913
+CEPH-83574917
```

Comment:
Hi @TejasC88 can we please include config file names as well..

Reply:
Done